Makefile

```diff
@@ -64,14 +78,42 @@ go-deps:
 	chmod +x ./vendor/k8s.io/code-generator/generate-groups.sh
 	chmod +x ./vendor/k8s.io/code-generator/generate-internal-groups.sh
 
-install-tools:
```

Comment:
I started to dig into writing this like:

```
TOOLS := setup-envtest go-junit-report golangci-lint
TOOLS_NEEDED := $(foreach tool,$(TOOLS),$(shell command -v go-junit-report > /dev/null)
```

But...honestly it's probably not worth the Make wizardary at this scale.  If we had 3 more tools to install it'd be worth reducing the copy paste, but IMO not yet.

Reply:
Yeah, I wasn't sure what the best pattern would be for this.